kyo-prelude/shared/src/test/scala/kyo/StreamTest.scala

```diff
@@ -481,6 +481,20 @@ class StreamTest extends Test:
             )
         }
 
+        "works with effects" in run {
```

Comment:
How about something like this?
```
        "non-empty stream" in {
            assert(
                Var.run(0) {
                    Stream.init(0 until 100).map(i => Var.update[Int](_ + i)).runDiscard.andThen(Var.get[Int])
                }.eval == 4950
            )
        }
```

Reply:
sounds good. Personally I'd prefer the expectation was compared to `(0 until 100).sum`, but I've gone with your suggestion for now.